### ISSUE TITLE
Test with Voiceover on on a real iOS device

### ISSUE THREAD

**Issue by dnfield** [flutter/flutter#42270]
We should be doing some integration testing on iOS with voiceover on, to make sure that certain things get selected or tha twe don't crash etc. when certain gestures/etc. are used. For example, it would enable a better regression test for https://github.com/flutter/engine/pull/12990.

AFAICT, this will require a dedicated device for it in the devicelab.  We can't do it on a simulator, and I don't know that there's any way to turn on voiceover in a test.  We could have a dedicated iOS device with voiceover enabled, and run some tests that way.

/cc @digiter @godofredoc @jonahwilliams @goderbauer @keyonghan

**Comment by jonahwilliams** [comment]
Accessibility support is important enough to dedicate a single device/machine to

**Comment by digiter** [comment]
What's the requirement for the device? Any iOS device?

**Comment by goderbauer** [comment]
Any device would do, it should probably run the latest iOS version.

**Comment by cyanglaz** [comment]
I'm also working on adding the accessibility bridge test to make sure the accessibility tree we generated is correct on iOS. see https://github.com/flutter/flutter/issues/42669
This will also require a dedicated iPhone in deviceLab to run the tests.

**Comment by godofredoc** [comment]
@dnfield is this something we can enable at runtime in the phone?

**Comment by dnfield** [comment]
There's no clear method for doing that at this time

**Comment by godofredoc** [comment]
@dnfield should we close this as infeasible?

**Comment by dnfield** [comment]
One suggestion has been to just hire some manual QA for this (@HansMuller @tvolkert  we've talked about this).
